lib/bolt/config/validator.rb

```diff
@@ -169,6 +195,18 @@ def validate(data, schema, location = nil)
         end
       end
 
+      # Adds an error that includes additional helpful information for values
+      # that accept multiple types.
+      #
+      private def multitype_error(message, value, definition)
+        if Array(definition[:type]).count > 1
+          types    = Array(definition[:type]) - [value.class]
+          message += " or must be of type #{types.join(', ')}"
```

Comment:
This feels pedantic, but what do you think about joining this with `or` instead? It looks like all multitypes take at most 2 types, and I feel like this can be misinterpreted as an "Array of Strings" or "Hash of strings":
```
  Value at 'modulepath' must be of type Array, String
```

Plus, I think having 3+ linked by "or" is like...annoying but still clear:
```
  Value at 'modulepath' must be of type Array or Hash or String
```

Reply:
Fine by me 🤷  Updated.

Just need to figure out why that one test is failing now...